packages/@uppy/provider-views/src/View.js

```diff
@@ -76,9 +76,9 @@ export default class View {
         body: {
           fileId: file.id,
         },
-        providerOptions: this.provider.opts,
         providerName: this.provider.name,
         provider: this.provider.provider,
+        [Symbol.for('requestClient')]: this.provider,
```

Comment:
What's the reasoning for using a symbol here? If it's to indicate this is an internal only property, wouldn't an underscore-leading property be more convenient?

Reply:
Iirc all properties on this object get JSON stringified so I had to prevent that for requestClient. Not the most elegant way but I couldn’t find another way to attach the request client instance to a file

Reply:
Can we add a comment with the reasoning for making this a Symbol then?

Reply:
A more ellegant way would be to make the property non-enumerable. Alternatively, could we export the symbol and pass it along instead of using `Symbol.for` everywhere? That would let us change it back to a string without causing a breaking change, and make the code easier to read

Reply:
made it non enumerable now